target_game.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun May 10 21:57:08 2020
+
+@author: Aly
+"""
+# Firstly I import packages that I might need
+
+from random import randint
+
+from psychopy.visual import Window, ImageStim
+from psychopy.event import getKeys, Mouse
+from psychopy.clock import Clock
+
+# I create a function which randomly changes the position of my target
+# I chose numbers depending on my DISP size (I want the target to be always presented inside the screen)
+# then the new position is set to the target, so the target can be moved
+
+def move_target_at_random_pos(target):
+    pos_x = randint(-700, 700)

```

Comment:
Maybe it would make sense to adjust the function a bit to ensure it works for different display sizes.
If I remember correctly from the Python course you can get the display size of each screen by importing: from win32api import GetSystemMetrics --> DISPSIZE = (GetSystemMetrics(0),GetSystemMetrics(1)).

Reply:
Amazing! Thank you so much :)

Reply:
If considering to change this, you should also adjust it in the move_target_at_random_pos function :).